### DISCUSSION TITLE
How to override the locales in the URL?

### DISCUSSION THREAD

**Discussion by thedaviddias** [vinissimus/next-translate#734] [General]
Hi everyone!

Do you any way to rename the locales in the URL? I'm using `fr-CA` and would like to just have `/fr/` in the url instead of `/fr-ca`.

Would you know any easy solution for this?

Thanks!

**Comment by thedaviddias** [comment]
@aralroca

**Reply by aralroca** [reply]
Yes, you can define the locales you want:

i18n.js
```js
module.exports = {
  locales: ['fr-CA', 'es-ES'],
  defaultLocale: 'fr-CA',
  pages: {
    '*': ['common'],
    '/404': ['error'],
    '/': ['home']
  }
}
```

**Reply by thedaviddias** [reply]
Thanks Aral, yeah yeah I know, but when I indicate `fr-CA`, in Next, it works when I access `localhost:3000/fr-ca` but not with `'localhost:3000/fr` only. Do you see what I mean?

**Reply by aralroca** [reply]
I don't understand what you want. You want to have both fr-CA and fr and have them share the same translations?
